package.json

```diff
@@ -484,6 +490,11 @@
                     "when": "view =~ /(azureResourceGroups|azureFocusView)/ && viewItem =~ /environmentVariablesItem/i",
                     "group": "1@1"
                 },
+                {
+                    "command": "containerApps.editEnvironmentVariableName",
+                    "when": "view =~ /(azureResourceGroups|azureFocusView)/ && viewItem =~ /environmentVariableItem/i",
+                    "group": "1@1"
```

Comment:
I think technically this should be 1@2

Reply:
I think 1@1 is correct, but it's confusing because the one above it is `environmentVariablesItem` with an s and they look super similar 😆